src/sentry/integrations/gitlab/integration.py

```diff
@@ -0,0 +1,210 @@
+from __future__ import absolute_import
+from time import time
+
+from six.moves.urllib.parse import urlparse
+from django.utils.translation import ugettext_lazy as _
+from django import forms
+
+from sentry import http
+from sentry.web.helpers import render_to_response
+from sentry.identity.pipeline import IdentityProviderPipeline
+from sentry.identity.gitlab.provider import GitlabIdentityProvider
+from sentry.integrations import Integration, IntegrationFeatures, IntegrationProvider, IntegrationMetadata
+from sentry.pipeline import NestedPipelineView, PipelineView
+from sentry.utils.http import absolute_uri
+
+
+DESCRIPTION = """
+Fill me out
```

Comment:
I think just copying bb is fine here: 
https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/bitbucket/integration.py#L18

Reply:
im probs just going to leave this for now and make a jira ticket

Reply:
Leave it for now if you can, my feature stuff is coming changes some things around this anyway